JS-103: JavaScript Functions/challenges/js/challenge2.js

```diff
@@ -0,0 +1,5 @@
+function getYear() {
+  const year = new Date().getFullYear();
```

Comment:
We can simplify this by immediately returning the value of year :
`return new Date().getFullYear();`